sdlib/d/gc/slab.d

```diff
@@ -23,6 +25,208 @@ struct SlabAllocGeometry {
 	}
 }
 
+struct SlabAllocInfo {
+private:
+	SlabAllocGeometry sg;
+	Extent* e;
+public:
+	bool allowsMetaData = false;
+	bool hasMetaData = false;
```

Comment:
Is this is POD (as in a bundle of data), or is this some sort of object (as in, the structs comes bundles with behavior that processes the data)?

It can't be both.